R/class_tgrid.R

```diff
@@ -107,41 +108,42 @@ as_deslist <- function(data, descol="ID") {
   structure(out, descol=descol)
 }
 
-##' Create a simtime object
-##'
-##' simtime objects allow the user to specify simulation start and end times, 
-##' along with the simulation time step.
-##'
-##' @rdname tgrid
-##' @name tgrid
-##'
-##' @param x tgrid object
-##' @param start simulation start time
-##' @param end simulation end time
-##' @param delta simulation time step
-##' @param add addition simulation times
-##' @param .offset the resulting set of times will be adjusted by this amount
-##' @param .scale the resulting set of times will be scaled by this factor
-##' @param ... passed on to other methods
-##' @export
-##' @examples
-##'
-##' peak <- tgrid(0,6,0.2)
-##' sparse <- tgrid(0,24,4)
-##'
-##' day1 <- c(peak,sparse)
-##'
-##' design <- c(day1, day1+72, day1+240)
-##' 
-##' \dontrun{
-##' mod <- mrgsolve::house()
-##'
-##' out <- mod %>% ev(amt=1000, ii=24, addl=10) %>% mrgsim(tgrid=design)
-##'
-##' plot(out,CP~., type='b')
-##' }
+#' Create a simtime object
+#'
+#' simtime objects allow the user to specify simulation start and end times, 
+#' along with the simulation time step.
+#'
+#' @rdname tgrid
+#' @name tgrid
```

Comment:
[ not changed with this PR ]

This is one of several spots that use both `@rdname` and `@name` with the same value. In the case, I think `rdname` is unnecessary.

Reply:
I think they can both go?  this documentation is associated with a function called `tgrid()`.